### ISSUE TITLE
[Bug]: Postfix operator on property access of arrow function call

### ISSUE THREAD

**Issue by p51lee** [swc-project/swc#6397]
### Describe the bug

`input.js` terminates without error but the output code
```js
// output.js
function(x) {
    return 0;
}().x--;
```
throws `SyntaxError`.

### Input code

```typescript
( x => 0 ) ( ) . x -- ;
```

### Playground link

https://play.swc.rs/?version=1.3.14&code=H4sIAAAAAAAAA9NQqFCwtVMwUNBU0ABiPSBXV1fBWgEAsCfsmRgAAAA%3D&config=H4sIAAAAAAAAA0WMTQrEIAxG75K1286id5hDBCctFv9IUhgR714tlu7C915ehUMsrBUyshCPS0pU%2FMMKZAOKZZcVTNf6tKEXagYUeScdiiyd%2BZSEJjUQXHRbGSWbQmYSeRHG3T9m66GQfucYKmjJdAc%2F0N7G%2FHPynaLySe0Cj9ke9LUAAAA%3D

### Expected behavior

Input code is terminated without error:
```console
$ node input.js # no error
```

### Actual behavior

Output code throws `SyntaxError`:
```console
$ node output.js
output.js:1
function(x) {
^^^^^^^^

SyntaxError: Function statements require a function name
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47

Node.js v18.11.0
```

### Version

1.3.14 (default SWC playground)

### Additional context

Note that this is a different issue with https://github.com/swc-project/swc/issues/6353.
https://github.com/swc-project/swc/issues/6353 is about a property access of `async` function without function call and this issue is about a property access of function call.

**Comment by magic-akari** [comment]
This issue has been fixed in #6359.
The new version is not released.

**Comment by kdy1** [comment]
I'll release a new version soon. Thank you @magic-akari
